gratipay/wireup.py

```diff
@@ -59,6 +59,11 @@ def db(env):
 
     return db
 
+def settings(db):
+    from gratipay.billing.exchanges import MINIMUM_CHARGE
+    db.run('DELETE FROM settings')
```

Comment:
Seems a little funky to have multiple web app processes potentially running this code at the same time. At the very least we should put this in a transaction so we don't try to delete from settings when there *is* no settings.